cluster-autoscaler/cloudprovider/oci/common/oci_tag.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2021-2024 Oracle and/or its affiliates.
+*/
+
+package common
+
+import (
+	oke "k8s.io/autoscaler/cluster-autoscaler/cloudprovider/oci/vendor-internal/github.com/oracle/oci-go-sdk/v65/containerengine"
+)
+
+// TagsGetter returns the oci tags for the pool.
+type TagsGetter interface {
+	GetFreeformTags(*oke.NodePool) (map[string]string, error)
```

Comment:
Since there will be two separate implementations of a `TagsGetter`, please rename this function to `GetNodePoolFreeformTags(*oke.NodePool) (map[string]string, error)`. 

Later, we can add a _second_ function to the interface called `GetInstancePoolFreeformTags(*core.InstancePool) (map[string]string, error)` and the associated implementation.

Reply:
Agreed, done.